docs/self-managed/optimize-deployment/configuration/system-configuration.md

```diff
@@ -130,74 +130,74 @@ at least one engine configured at all times. You can configure multiple engines
+| engines.${engineAlias}.rest                    | http://localhost:8080/engine-rest | A base URL that will be used for connections to the Camunda Engine REST API.                                                                                                                                                                                                                                                                                                                                                                            |
+| engines.${engineAlias}.importEnabled           | true                              | Determines whether this instance of Optimize should import definition & historical data from this engine.                                                                                                                                                                                                                                                                                                                                               |
+| engines.${engineAlias}.eventImportEnabled      | false                             | Determines whether this instance of Optimize should convert historical data to event data usable for event based processes.                                                                                                                                                                                                                                                                                                                             |
+| engines.${engineAlias}.authentication.enabled  | false                             | Toggles basic authentication on or off. When enabling basic authentication, please be aware that you also need to adjust the values of the user and password.                                                                                                                                                                                                                                                                                           |
+| engines.${engineAlias}.authentication.user     |                                   | When basic authentication is enabled, this user is used to authenticate against the engine.<br /><br /> Note: when enabled, it is required that the user has <ul><li>READ & READ*HISTORY permission on the Process and Decision Definition resources</li> <li>READ permission on \_all* ("\*")Authorization, Group, User, Tenant, Deployment & User Operation Log resources</li></ul> to enable users to log in and Optimize to import the engine data. |
```

Comment:
@christinaausley the build broke on this changed line, prettier had a hard time formatting the asterisks and TBH it didn't do a great job. 

I intend to fix it, but I want to make sure I'm fixing it correctly. Do you know what the `("*")` on this line is intended to indicate? It seems like a strange outlier that I think I might be able to remove. 

For reference, [here is the section on the live site](https://docs.camunda.io/docs/self-managed/optimize-deployment/configuration/system-configuration/#connection-to-camunda-platform-7). The line (and `(*)`) in question looks like this: 

![image](https://user-images.githubusercontent.com/1627089/171275183-b23967af-44b8-4235-8f09-525dfe73e03b.png)

Reply:
I think this is about Optimize and maybe @megglos or @RomanJRW can better answer this question

Reply:
For the sake of moving forward, I updated this line so that it _almost_ matches what's currently in production, and doesn't confuse prettier. 

It turns out prettier was _really_ getting confused by the underscore in READ_HISTORY, resulting in weird things happening with the rest of the line. In https://github.com/camunda/camunda-platform-docs/pull/928/commits/4763b2f41396dd1c32cf849823c9b5cd2cbb38d5, I surrounded READ_HISTORY with backticks to turn it into a code block, so that prettier doesn't touch it. I also surrounded READ with backticks, for consistency. 

With the code blocks added, this item now looks like this: 

<img width="306" alt="image" src="https://user-images.githubusercontent.com/1627089/171930431-fc241a57-6b69-4c5e-b2f7-e0c80302beb5.png">

My hope is that we can merge this as-is, and if there are changes that come up from @megglos or @RomanJRW I will add them in a follow-up PR. 

@christinaausley would you mind reviewing again to verify these changes conform to our writing guidelines?